### ISSUE TITLE
open browser in background

### ISSUE THREAD

**Issue by aacp29** [Ranchero-Software/NetNewsWire#4069]
I am under OSX Ventura 13.5 with NetNewsWire 6.1.4 (6120) and I can't open a page in background, with the option checked or not and with the shift key press or not.

**Comment by aacp29** [comment]
Yes, and I’ve just tried with the new Arc browser : it’s the same !

**Comment by brentsimmons** [comment]
Unfortunately this is a bug with Chromium browsers.
